data/bash-completion/fwupdtool

```diff
@@ -13,6 +13,8 @@ _fwupdtool_cmd_list=(
 	'firmware-parse'
 	'firmware-sign'
 	'firmware-patch'
+	'fix-security'
+	'unfix-security'
```

Comment:
I think we want `fwupdmgr` and `fwupdtool` to have consistent commands.

Reply:
Oopa, fixed, thanks.